doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -49,17 +29,16 @@ sdk\<service name>\<package name>\<package name>.sln
 - `<service name>` - Should be the short name for the azure service. e.g. deviceupdate. It is usually your service name in REST API specifications. For instance, if the API spec is under `specification/storage`, the service would normally be `storage`.
 - `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
 
-### Use Cadl as Input API spec
 
-#### Before you Start
+### Before you Start
   
-  Make sure you have cadl project in [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow [Cadl Getting Start](https://github.com/microsoft/typespec/#using-node--npm) to initialize your Cadl project, and refer to [Cadl Structure Guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md) to configure your Cadl project.
+  Make sure you have typespec project in [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow [Typespec Getting Start](https://github.com/microsoft/typespec/#using-node--npm) to initialize your typespec project, and refer to [Typespec Structure Guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md) to configure your typespec project.
```

Comment:
```suggestion
  Make sure you have typespec project in [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow [TypeSpec Getting Started](https://github.com/microsoft/typespec/#using-node--npm) to initialize your typespec project, and refer to [TypeSpec Structure Guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md) to configure your typespec project.
```

Also did you mean to make the getting started link to the node/npm anchor link?

Reply:
updated